docs/advanced/document-store/qdrant.md

```diff
@@ -205,57 +213,4 @@ Embeddings Nearest Neighbours with "price" at most 7:
 	embedding=[6. 6. 6.],	 price=6
 	embedding=[5. 5. 5.],	 price=5
 	embedding=[4. 4. 4.],	 price=4
-```
-### Example of `.filter` with a filter
```

Comment:
Why is this removed?

Reply:
Fixed. It was an merging conflict handling error.